utils/execTo.js

```diff
@@ -0,0 +1,35 @@
+const {
+  execSync
+} = require(`child_process`);
+
+const execToStr = (cmd) => {
+  try {
+    var _execSync;
+
+    return String((_execSync = execSync(cmd, {
+      encoding: `utf8`
+    })) !== null && _execSync !== void 0 ? _execSync : ``).trim();
```

Comment:
This is pretty hard to follow. I previously took the time to break down what's going on here, but my comment was removed (perhaps due to a force push or something to do with Github).

Can we break this out into steps and add some comments? I'm happy to do this as a pair / group.

Reply:
Ye not sure why some of the comments were removed, i didn't do a force push. I tried your suggested method but it didn't work so it would be good to run through this again